projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -1,36 +1,52 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Main class file for Jetpack Admin pages.
+ *
+ * @package automattic/jetpack
+ */
 
 use Automattic\Jetpack\Identity_Crisis;
 use Automattic\Jetpack\Redirect;
 use Automattic\Jetpack\Status;
 
-// Shared logic between Jetpack admin pages
+/**
+ * Shared logic between Jetpack admin pages.
+ */
 abstract class Jetpack_Admin_Page {
-	// Add page specific actions given the page hook
-	abstract function add_page_actions( $hook );
+	/** Add page specific actions given the page hook.
```

Comment:
We normally don't start the summary on the same line as the `/**`.
```suggestion
	/**
	 * Add page specific actions given the page hook.
```